### ISSUE TITLE
Accessibility: Keyboard Navigation/Focus

### ISSUE THREAD

**Issue by deinhofer** [studyathome-internationally/hub#7]
When testing the keyboard navigation of the hub (landing page) there are some accessibility issues:

1. Anchors are not visible (only when hovered)
2. Anchors and course boxes don't have clearly visible border when focused

![image](https://user-images.githubusercontent.com/4621810/87136085-40235c00-c29b-11ea-8ade-fd137f63c038.png)

**Comment by deinhofer** [comment]
This is also the case for other pages.

**Comment by sabicalija** [comment]
I've noticed this also before. This is the default behavior, however.

**Comment by deinhofer** [comment]
If the anchor is only visible when hovering, it's ok I think. But if it's focused it must have the visible border.

Also on a course details page the focus order and visualization is not good.
